### PR TITLE
Add participants graph

### LINKED ISSUE #97: Analysis UI: Add trends over time
Once #95 and #96 are resolved, we should:

* Choose a plotting library
* Show CIs over time
* Show participant counts over time

### PR DESCRIPTION
**Pull request by jessie-ross**
**This PR adds a participants graph.**
- Full width for now under the metric graphs
- Includes: Totals, Not Final, Variations
- Not sure it belongs here, above the metrics list? Different metrics have different notFinal counts?

For a screenshot see: pbmo2S-vR-p2#comment-1156

Resolves #97 

## How has this been tested?

- Automated tests that cover added/changed functionality
- Manual testing with production data (locally)

### CODE REVIEW

**Review by yanirs — CHANGES_REQUESTED** [review]
As noted on the P2 thread, let's make it a single graph at the top of the page, based on the counts from the primary metric. We don't need to plot the not final counts as they're only interesting on a per-metric level for the last analysis date – it’s an indication of how much the results may change.

**Comment by aaronyan** [comment]
Looking great! +1 to removing `Not Final` from the participants graph. I would also remove the `Total` series, because I think the participants graphs is mainly comparing the two variations.

**Comment by jessie-ross** [comment]
## Update

All the feedback has hopefully been taken on, see pbmo2S-vR-p2#comment-1168 for more info.

File: components/experiment-results/CondensedLatestAnalyses.tsx [hunk 1]
```diff
@@ -30,6 +31,7 @@ import {
   MetricParameterType,
 } from '@/lib/schemas'
 import * as Variations from '@/lib/variations'
+import * as Visualisations from '@/lib/visualisations'
```
Comment (yanirs) [inline components/experiment-results/CondensedLatestAnalyses.tsx:34]: Let's stick to American spelling for consistency (`visualizations`). :slightly_smiling_face:

**Comment by yanirs** [comment]
Loading the result page on Vercel yields an error: https://abacus-git-add-participant-graph.a8c.vercel.app/experiments/2/results. Is this a general problem or specific to the Swagger API results?

**Review by withinboredom — APPROVED** [review]
LGTM, just needs the renaming to US-EN (which I didn't even notice).

**Comment by jessie-ross** [comment]
> Loading the result page on Vercel yields an error: https://abacus-git-add-participant-graph.a8c.vercel.app/experiments/2/results. Is this a general problem or specific to the Swagger API results?

Yeah this is pretty minor to cover for either way. 

> Let's stick to American spelling for consistency (visualizations). 🙂

Sure! 🇺🇸 

---

I'm just holding of for Aarons feedback on the total.

**Comment by yanirs** [comment]
Looking at https://abacus-git-add-participant-graph.a8c.vercel.app/experiments/1/results now that it's fixed, it's a bit confusing with a single day of data, but I'll open a separate issue.

**Review by yanirs — APPROVED** [review]

**Pull request merged**